src/pages/ProAmmPools/index.tsx

```diff
@@ -254,24 +254,12 @@ export default function ProAmmPoolList({ currencies, searchValue, isShowOnlyActi
     ) : null
   }
 
-  const ITEM_PER_PAGE = 5
+  const ITEM_PER_PAGE = 2
```

Comment:
this is only for testing, right?
Please revert it to 5

Reply:
Done.